src/Microsoft.Identity.Client/OAuth2/OAuthConstants.cs

```diff
@@ -83,6 +83,59 @@ internal static class OAuth2Error
         public const string AuthorizationPending = "authorization_pending";
     }
 
+    internal static class OAuth2SubError
+    {
+        /// <summary>
+        /// Condition can be resolved by user interaction during the interactive authentication flow.
+        /// </summary>
+        public const string BasicAction = "basic_action";
```

Comment:
Unfortunately I don't know how to "repro" each of these, so I couldn't test E2E. Still working on this, but unit tests should give some confidence.